processor/servicegraphprocessor/processor.go

```diff
@@ -400,7 +397,7 @@ func buildDimensions(e *store.Edge) pcommon.Map {
 func (p *serviceGraphProcessor) buildMetrics() (pmetric.Metrics, error) {
 	m := pmetric.NewMetrics()
 	ilm := m.ResourceMetrics().AppendEmpty().ScopeMetrics().AppendEmpty()
-	ilm.Scope().SetName("traces_service_graph_servicegraphprocessor")
+	ilm.Scope().SetName("traces_service_graph")
```

Comment:
Maybe? I'm not entirely sure. This property is not used in the Grafana visualization, nor in the spec. So I don't think it'll have an impact.

Also, I think they should be renamed anyway since we're moving the processor to a connector.

Reply:
I guess my question would be: do we expect people to be using this metric when consuming data from this component? The public API for this component would include the metrics it generates, IMO. If we don't expect users to be consuming this metric on their backends, I'm OK in merging this as is, otherwise, we should probably come up with a plan for not breaking current users without notice.

Reply:
> do we expect people to be using this metric when consuming data from this component?

No. The data is very difficult to consume without the Grafana visualization. I don't think anyone is checking the metrics directly.